src/App.tsx

```diff
@@ -5,18 +5,38 @@ import LibraryPage from "./pages/Library";
 import RootLayout from "./pages/Root";
 import ErrorPage from "./pages/Error";
 import PlayListPage from "./pages/PlayListPage";
-
-import { useState, useEffect } from "react";
+import { useState, useEffect, createContext } from "react";
 import Login from "./components/Login";
 import localStorageToken from "./helpers/localStorageToken";
-import getTokenFromUrl from "./helpers/getTokenFromUrl";
+
+import SpotifyWebApi from "spotify-web-api-js";
+import getAuthParamsFromHash from "./helpers/getAuthParamsFromHash";
+const spotifyApi = new SpotifyWebApi();
+
+type StateType = {
+  songOnPlayer: string[];
+  spotifyApi: SpotifyWebApi.SpotifyWebApiJs;
+  playSelectedSong: (selectedSong: string[]) => void;
```

Comment:
I'm confused because it's called "selected song" (singular) but accepts a list of strings as argument.